### PR TITLE
in read.quitte, allow x2020 as period and wrong default column order

### PR DESCRIPTION
**Pull request by orichters**
in `read.quitte`, be a bit more tolerant while reading csv/mif files:
- Allow period column names that read like `X2020` with a single character before the year, which is apparently done in the IIASA universe sometimes, to make it a string maybe? Delete the character afterwards.
- allow non-standard sorting of default columns, but fix the sorting afterwards.
- if default columns are missing, tell the user which ones.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "title": "quitte: Bits and pieces of code to use with quitte-style data frames",
-  "version": "0.3106.0",
+  "version": "0.3106.1",
```
Comment (0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q) [inline .zenodo.json:3]: Not really a bugfix, eh?

**Review by 0UmfHxcvx5J7JoaOhFSs5mncnisTJJ6q — APPROVED** [review]
> Allow period column names that read like X2020 with a single character before the year, which is apparently done in the IIASA universe sometimes, to make it a string maybe?

`read.csv()` does that.
```
> colnames(read.csv2(system.file('extdata', 'REMIND_generic_r7552c_1p5C_Def-rem-5.mif', package = 'quitte')))
 [1] "Model"    "Scenario" "Region"   "Variable" "Unit"     "X2005"    "X2010"    "X2015"    "X2020"   
[10] "X2025"    "X2030"    "X2035"    "X2040"    "X2045"    "X2050"    "X2055"    "X2060"    "X2070"   
[19] "X2080"    "X2090"    "X2100"    "X2110"    "X2130"    "X2150"    "X"       
```
Sure this comes from IIASA?

File: .zenodo.json (hunk 1 above)
Comment (orichters) [reply .zenodo.json]: right, changed it to 0.3107.0 instead…

**Comment by orichters** [comment]
> Sure this comes from IIASA?

No, probably not their invention, but I noticed it in a file that was used in the data exchange with them.

**Pull request merged**